console/src/app/services/interceptors/exhausted.http.interceptor.ts

```diff
@@ -13,7 +13,7 @@ export class ExhaustedHttpInterceptor implements HttpInterceptor {
 
   intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     // Don't check the cookie for the environment.json
-    let cookie$ = req.url.endsWith('/assets/environment.json') ? of(undefined) : this.exhaustedSvc.checkCookie();
+    let cookie$ = req.url.endsWith('./assets/environment.json') ? of(undefined) : this.exhaustedSvc.checkCookie();
```

Comment:
So is the idea that the cookie references to the environment.json? and that again to the instance url?

Reply:
fixed it